### PR TITLE
OS: 11478866 adding a probe_stack to one of GetPrototypeSpecial API.

### PR DESCRIPTION
**Pull request by akroshg**

### CODE REVIEW

File: lib/Runtime/Library/JavascriptProxy.cpp [hunk 1]
```diff
@@ -1451,6 +1451,8 @@ namespace Js
 
     RecyclableObject* JavascriptProxy::GetPrototypeSpecial()
     {
+        PROBE_STACK(GetScriptContext(), Js::Constants::MinStackDefault);
```
Comment (agarwal-sandeep) [inline lib/Runtime/Library/JavascriptProxy.cpp]: Do we have a test?

Comment (akroshg) [inline lib/Runtime/Library/JavascriptProxy.cpp]: I have a test but that is something exhausting the stack. Pretty much revealing that probe_stack is missing. I don't see a value in it. What do you think?

Comment (agarwal-sandeep) [inline lib/Runtime/Library/JavascriptProxy.cpp]: I was interested a small snippet, if its huge ignore it

**Comment by akroshg** [comment]
@dotnet-bot test Windows x64_release please

**Comment by akroshg** [comment]
@dotnet-bot test Windows x64_release

File: lib/Runtime/Library/JavascriptProxy.cpp (hunk 1 above)
Comment (agarwal-sandeep) [inline lib/Runtime/Library/JavascriptProxy.cpp]: Have a question related to this. JavascriptProxy::DeleteProperty does have PROBE_STACK but why SetProperty doesn't? Similarly IsExtensible which seems to have same pattern of CallFunction?

Comment (akroshg) [inline lib/Runtime/Library/JavascriptProxy.cpp]: SetProperty calls GetOwnPropertyDescriptor, which eventually has it. but I don't have test case to back that up. I can add the check there as well. 
I didn't understand the IsExtensible question.

Comment (agarwal-sandeep) [inline lib/Runtime/Library/JavascriptProxy.cpp]: discussed offline, Akrosh will check all proxy apis and make sure we have stack probe where needed

**Pull request merged**